### PR TITLE
chore(humanTask): update GET request description

### PR DESCRIPTION
**Pull request by Shmayro**
Add missing filters to the description:
- user_id
- show_assigned_to_others

closes [RUNTIME-1309](https://bonitasoft.atlassian.net/browse/RUNTIME-1309)

### CODE REVIEW

**Comment by Shmayro** [comment]
bonita-doc related PR https://github.com/bonitasoft/bonita-doc/pull/2087

File: openapi/paths/API@bpm@humanTask.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ get:
 
     - can order on `name`, `priority`, `dueDate`, `state`, `processDefinitionId`, `processInstanceId`, `parentActivityInstanceId`, `assigneeId`, `parentContainerId`, `displayName`, `reachedStateDate`
     - can search on any field that can be used to order results
-    - can filter on `assigned_id`, `state`, `name`, `displayName`, `processDefinitionId`, `caseId`, `rootProcessInstanceId`, `parentProcessInstanceId`
+    - can filter on `assigned_id`, `user_id`, `show_assigned_to_others`, `state`, `name`, `displayName`, `processDefinitionId`, `caseId`, `rootProcessInstanceId`, `parentProcessInstanceId`
```
Comment (rbioteau) [inline openapi/paths/API@bpm@humanTask.yaml:10]: Is it true in all supported version ? (7.11 to today)

Comment (Shmayro) [reply openapi/paths/API@bpm@humanTask.yaml:10]: The filter **show_assigned_to_others** has been just introduced for the 2022.2 (https://github.com/bonitasoft/bonita-web-sp/pull/903)
but the **user_id** was existing (7.11 included)

Comment (rbioteau) [reply openapi/paths/API@bpm@humanTask.yaml:10]: It should be displayed somewhere then (eg: since 2022.2) for `show_assigned_to_others`

Comment (Shmayro) [reply openapi/paths/API@bpm@humanTask.yaml]: done

**Pull request merged**